webpack.config.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
     // WebpackDevServer host and port:
     // server: 'webpack-dev-server/client?http://localhost:8080',
     signup: "./src/entry-points/signup",
-    dashboard: "./src/entry-points/dashboard"
+    dashboard: "./src/entry-points/dashboard",
+    login: "./src/entry-points/login",
```

Comment:
1. How would the current app be affected by a having a fully generic solution:  
- one generic entry-point here in the `webpack.config`
- a generic `src/entry-point` file that points to a generic `AppLoader_container` file
- And loads a generic `AppLoader` component

2. How would the current app be affected by having a semi-generic solution: 
- (I think I have touched on this in the entry-point file) Would we be able to use info from this object as a condition in a generic `src/entry-points` file? 

So if it's looking for `login:` here > it's pointed to a generic file > that at the bottom uses the `login` as info to serve `init_app(document.getElementById("root"), <LoginMainContainer />)`